build/templates/_library_singleton.py.mako

```diff
@@ -55,5 +55,17 @@ def get():
             except OSError:
                 raise errors.DriverNotInstalledError()
             _instance = _library.Library(ctypes_library)
+            try:
+                runtime_env_ctype = ctypes.create_string_buffer(platform.python_implementation().encode('windows-1251'))  # case C010
+                version_ctype = ctypes.create_string_buffer(platform.python_version().encode('windows-1251'))  # case C010
+                reserved_ctype = ctypes.create_string_buffer(''.encode('windows-1251'))  # case C010
+                _instance.${config['c_function_prefix']}SetRuntimeEnvironment(
+                    runtime_env_ctype,
+                    version_ctype,
+                    reserved_ctype,
+                    reserved_ctype
```

Comment:
@marcoskirsch would it be considered best practice to have reserved1_ctype, reserved2_ctype or do you think it won't matter?

Reply:
Doesn't matter I think.